Use JSON::MaybeXS

labrown opened the pull request:
Use JSON::MaybeXS instead of the buggy, action at a distance causing, "use JSON -convert_blessed_universally".

See #3.

labrown committed:
Use JSON::MaybeXS

renormalist commented:
Thanks. Give me a second to think about it - is there a lazy loading "require" form for JSON::MaybeXS which avoids the compile time "use"?

labrown commented:
I'll see if it works as a require when I get a chance.

labrown committed:
require JSON instead of using eval magic

Just "require JSON" instead of using eval magic with
'-convert_blessed_universally' which is considered a
pretty bad source of "spooky action at a distance".

labrown commented:
Updated PR to juse <code>require JSON</code> instead of using eval magic with -convert_blessed_universal_mode.  This passes the tests on my system and doesn't cause problems with possible spooky action at a distance.
